platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/offline/OfflineManager.java

```diff
@@ -114,6 +116,7 @@ public void run() {
     }).start();
   }
 
+  @UiThread
```

Comment:
@tobrun Is it currently possible to get the manager instance from the main thread but then call the other methods in a separate thread?